Bottom: ce8404676d1b873e1f016d99b4884b22472c8071
Top:    913366d6445de6b67c446e7f6a8faae1f5a0e682
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 01:13:42 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
